test/test_files/transaction/create_rel/insert_rels_to_small_list.test

```diff
@@ -0,0 +1,84 @@
+-GROUP CreateRelTest_insertRelsToSmallList
+-DATASET CSV rel-update-tests
+--
+
+-DEFINE_STATEMENT_BLOCK INSERT_RELS_TO_SMALL_LIST [
+-STATEMENT MATCH (a:person),(b:person) WHERE a.ID=1 and b.ID=300 CREATE (a)-[e:knows{length: 300, place: 'this is a long str', tag: ['123', 'good']}]->(b)
+---- ok
+-STATEMENT MATCH (a:person),(b:person) WHERE a.ID=1 and b.ID=700 CREATE (a)-[e:knows{length: 45, tag: ['123', 'good']}]->(b)
+---- ok
+-STATEMENT MATCH (a:person),(b:person) WHERE a.ID=1 and b.ID=400 CREATE (a)-[e:knows{length: 34, place: 'short'}]->(b)
+---- ok
+-STATEMENT MATCH (a:person),(b:person) WHERE a.ID=1 and b.ID=600 CREATE (a)-[e:knows{}]->(b)
+---- ok
+]
+
+#insertRelsToSmallListCommitNormalExecution
+-CASE insertRelsToSmallListCommitNormalExecution
+-LOG insertRelsToSmallListCommitNormalExecution
+
+-STATEMENT BEGIN WRITE TRANSACTION
+---- ok
+
+-INSERT_STATEMENT_BLOCK INSERT_RELS_TO_SMALL_LIST
+-STATEMENT COMMIT
+---- ok 
+-STATEMENT BEGIN WRITE TRANSACTION
+---- ok
+-STATEMENT MATCH (a:person)-[e:knows]->(b:person) where b.ID > 200 AND a.ID = 1 RETURN e.length, e.place, e.tag
+---- 4
+300|this is a long str|[123,good]
+34|short|
+45||[123,good]
+||
+
+#insertRelsToSmallListCommitRecovery
+-CASE insertRelsToSmallListCommitRecovery
+-LOG insertRelsToSmallListCommitRecovery
+
+-STATEMENT BEGIN WRITE TRANSACTION
+---- ok
+-INSERT_STATEMENT_BLOCK INSERT_RELS_TO_SMALL_LIST
+-STATEMENT COMMIT_SKIP_CHECKPOINT
+---- ok 
+-RELOADDB
+-STATEMENT BEGIN WRITE TRANSACTION
+---- ok
+-STATEMENT MATCH (a:person)-[e:knows]->(b:person) where b.ID > 200 AND a.ID = 1 RETURN e.length, e.place, e.tag
+---- 4
+300|this is a long str|[123,good]
+34|short|
+45||[123,good]
+||
+
+
+
+#insertRelsToSmallListRollbackNormalExecution
```

Comment:
Let's remove this comment for all tests. They seem to be duplicate with the CASE name.

Also I think we probably also don't need LOG after CASE. LOG is useful when we try to differentiate multiple statements in a single CASE (which is more common in read only tests). @ray6080 can comment this